### DISCUSSION TITLE
Does the PICO come attached with this board?

### DISCUSSION THREAD

**Discussion by rgfx** [phil-barrett/PicoCNC#3] [Q&A] [answered]
On the image on Tindie am seeing the PICO attached, do I buy the PICO separately?

**Comment by phil-barrett** [comment, marked as answer]
Yes, you need to purchase a Pico separately.  Thanks for asking, I will change the Tindie page to be very clear on that.

On a related question, I have though about carrying Picos but there is not much of volume discount so I would have to charge above the list price to cover my costs (shipping, repackaging, inventory risk, returns,...). Do you think that would be perceived as bad by customers?

**Comment by rgfx** [comment]
I personally wouldn't mind buying a bundled Pico. I live in Costa Rica and plan on buying  two of your boards once available. Less paperwork/shipping/taxes for me.
